proto/v1beta1/vulnerability.proto

```diff
@@ -89,6 +89,38 @@ message Vulnerability {
     // obsolete details.
     bool is_obsolete = 9;
   }
+
+  // Windows details get their own format because the information format and model
+  // don't match a normal detail. Specifically windows updates are done as patches,
+  // thus windows vulnerabilities really are a missing package, rather than a package
+  // being at an incorrect version.
+  repeated WindowsDetail windows_details = 4;
+
+  message WindowsDetail {
+    // Required. The CPE URI in
+    // [cpe format](https://cpe.mitre.org/specification/) in which the
+    // vulnerability manifests. Examples include distro or storage location for
+    // vulnerable jar.
+    string cpe_uri = 1;
+
+    // The name of the vulnerability.
+    string name = 2;
+
+    // The description of the vulnerability.
+    string description = 3;
+
+    // Required. The names of the KBs which have hotfixes to mitigate this vulnerability.
+    // Note that there may be multiple hotfixes (and thus multiple KBs) that mitigate
+    // a given vulnerability.
+    repeated KnowledgeBase kbs = 4;
+
+    message KnowledgeBase {
+      // The KB name (generally of the form KB1234)
```

Comment:
Is it always 4 digits? Or is the idea that KB is followed by digits?

Reply:
It's not always 4 digits (and as the number of KBs goes up it will continue to gain digits).

Reply:
Gotcha. Maybe then modify this to be KB[0-9]+ or something to prevent the reader from making incorrect assumptions? I'm open to ideas here for how to clarify.

Reply:
Updated comment to include regex.